demos/browser/demo.js

```diff
@@ -1,6 +1,8 @@
 /* global tus */
 /* eslint-disable no-console, no-alert */
 
+'use strict'
```

Comment:
Why do some files get strict mode but not all?

Reply:
See https://tc39.es/ecma262/#sec-strict-mode-code: 

> [Module code](https://tc39.es/ecma262/#sec-types-of-source-code) is always strict mode code.

That means, only non-ESM files need the `'use strict'` directive.

Reply:
I see, that you!